pyproject.toml

```diff
@@ -1,18 +1,22 @@
 [tool.poetry]
+package-mode = false
 name = "arena"
 version = "0.1.0"
 description = ""
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.12"
+python = "^3.11"
```

Comment:
I changed it to avoid `module 'pkgutil' has no attribute 'ImpImporter'. Did you mean: 'zipimporter` error
ref: https://stackoverflow.com/questions/77364550/attributeerror-module-pkgutil-has-no-attribute-impimporter-did-you-mean